app/+not-found.tsx

```diff
@@ -1,8 +1,8 @@
 import { Link, Stack } from 'expo-router';
 import { StyleSheet } from 'react-native';
 
-import { ThemedText } from '@/components/ThemedText';
 import { ThemedView } from '@/components/ThemedView';
+import { ThemedText } from '@/components/ui/text/ThemedText';
```

Comment:
```suggestion
import { ThemedText } from '@/components/ui/typography/ThemedText';
```
نخلي الpath يكون general اكتر؟

Reply:
فكرة ممتازة 😉